test/tests/functional/pbs_qstat.py

```diff
@@ -57,6 +58,75 @@ def parse_json(self, dictitems, qstat_attr):
                         self.parse_json(val, qstat_attr)
         return qstat_attr
 
+    def validate_el(self, attr_dict, user):
```

Comment:
Just a suggestion take it leave it up to you: For more readability I suggest to change this method name to validate_attribute, what is `el`? 😄